rust/src/lib/nispor/route_rule.rs

```
@@ -61,6 +63,13 @@ pub(crate) fn get_route_rules(
             if !protocols.contains(rule_protocol) {
                 continue;
             }
+            // We only support modifying local rule from kernel, the others
+            // should be ignored.
```

Just to be sure, should we silently ignore other rules from kernel or raise the exception ?

---

This is the nispor code, we cannot raise an exception if we get a rule with RouteProtocol::Kernel. If we do that, we will always fail as we cannot control rules that are set/handled by kernel.

As we did in the past, we silently ignore them.